File: day1.js
function countIncreases(str) {
    let arr = str.split("\n");
    let count = 0;
    for (let i = 1; i < arr.length - 1; i++) {
        if (arr[i] > arr[i - 1]) {
            count++;
        }
    }
    console.log(count);
}

let str = window.prompt("Enter input: ");

File: day1.py
def countIncreases(arr):
    count = 0;
    for i in range (1, len(arr)):
        if arr[i - 1] < arr [i]:
            count += 1
    return count
def main():
    inputL = '''\
100
'''
    arr = [int(line) for line in inputL.splitlines()]
    print(countIncreases(arr))
if __name__ == "__main__":
    main()